engine/src/main/java/org/hibernate/validator/internal/engine/constraintvalidation/ClassBasedValidatorDescriptor.java

```diff
@@ -25,28 +25,38 @@
  * Represents an implementation of {@link ConstraintValidator}.
  *
  * @author Gunnar Morling
+ * @author Guillaume Smet
  */
 class ClassBasedValidatorDescriptor<A extends Annotation> implements ConstraintValidatorDescriptor<A> {
 
-	private static final long serialVersionUID = -8207687559460098548L;
-	private static final Log LOG = LoggerFactory.make( MethodHandles.lookup() );
+	private static final Log log = LoggerFactory.make( MethodHandles.lookup() );
```

Comment:
Why the change in casing, it's static after all?

Reply:
Raaah, I'm in Search mode when they do it the opposite way and are fixing it the opposite way. Will fix.